pkg/auth/handler/welcome_email.go

```diff
@@ -0,0 +1,117 @@
+package handler
+
+import (
+	"encoding/json"
+	"net/http"
+
+	"github.com/skygeario/skygear-server/pkg/auth"
+	"github.com/skygeario/skygear-server/pkg/auth/dependency/welcemail"
+	"github.com/skygeario/skygear-server/pkg/core/auth/authz"
+	"github.com/skygeario/skygear-server/pkg/core/auth/authz/policy"
+	"github.com/skygeario/skygear-server/pkg/core/handler"
+	"github.com/skygeario/skygear-server/pkg/core/inject"
+	"github.com/skygeario/skygear-server/pkg/core/server"
+	"github.com/skygeario/skygear-server/pkg/server/skyerr"
+)
+
+// AttachWelcomeEmailHandler attaches WelcomeEmailHandler to server
+func AttachWelcomeEmailHandler(
+	server *server.Server,
+	authDependency auth.DependencyMap,
+) *server.Server {
+	server.Handle("/welcome_email/test", &WelcomeEmailHandlerFactory{
```

Comment:
May I know when will us use this handler? Or it is for test only.

Reply:
This is for portal user to test before they confirm and save the setting.

Reply:
https://github.com/SkygearIO/forgot_password/blob/master/forgot_password/handlers/welcome_email.py#L126

Reply:
Thanks.